Domains/2-DisciplinePhysical/Civil/CivilPhysical/CivilPhysical.ecschema.xml

```diff
@@ -48,6 +48,21 @@
             <Class class="CurbType"/>
         </Target>
     </ECRelationshipClass>
+    <ECEntityClass typeName="RetainingWall" displayLabel="Retaining Wall" description="A bis:PhysicalElement intended to retain soil or other materials.">

```

Comment:
Proposed description:

A lr:LinearPhysicalElement modeling a structure built to hold back soil or other materials, supporting them laterally, preventing them from sliding or eroding away.